lacework-agent/templates/daemonset.yaml

```diff
@@ -72,12 +75,17 @@ spec:
         volumeMounts:
           - name: cfgmap
             mountPath: /var/lib/lacework/config
+{{- if .Values.cloudservice.gke.autopilot }}
```

Comment:
I am not sure how easy it is to not include the GKE Autopilot support as part of the 6.0.1 release. We don't not support GKE Autopilot yet and I just worry that it will confuse customers and SEs.

Reply:
I can remove this, thanks!